README.md

```diff
@@ -45,15 +45,64 @@ strategy:
     exclude:
       - { os: windows-2019, php: "8.1" }
       - { os: windows-2019, php: "8.0" }
-      - { os: windows-2019, php: "7.4" }
-      - { os: windows-2019, php: "7.3" }
+      - { os: windows-2022, php: "7.4" }
+      - { os: windows-2022, php: "7.3" }
       - { os: windows-2022, php: "7.2" }
       - { os: windows-2022, php: "7.1" }
 ```
 
 Currently, `windows-2019` may be used for all PHP versions, although this may
 change in future releases.
 
+### Manually Installing Toolsets
+
+It is possible to manually install older toolsets on `windows-2022` using an
+approach suggested in [actions/runner-images#9701](https://github.com/actions/runner-images/issues/9701).
+The following example installs VC15 by its
+[Component ID](https://learn.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-build-tools?view=vs-2022)
+to allow building PHP 7.2, 7.3, and 7.4 on a `windows-2022` image:
+
+```yml
+run:
+  steps:
+    - name: Delete components
+      shell: pwsh
+      run: |
+              Set-Location "C:\Program Files (x86)\Microsoft Visual Studio\Installer\"
+              $InstallPath = "C:\Program Files\Microsoft Visual Studio\2022\Enterprise"
+              $componentsToRemove= @(
+                "Microsoft.VisualStudio.Component.VC.v141.x86.x64"
+              )
+              [string]$workloadArgs = $componentsToRemove | ForEach-Object {" --remove " +  $_}
+              $Arguments = ('/c', "vs_installer.exe", 'modify', '--installPath', "`"$InstallPath`"",$workloadArgs, '--quiet', '--norestart', '--nocache')
+              # should be run twice
```

Comment:
Care to clarify why it needs to run twice? :-)

Reply:
You'd have to ask the OP in https://github.com/actions/runner-images/issues/9701.

While experimenting with a solution on PHPC, I did get a successful build with removing the duplicate `Start-Process` line _as well as_ the first step that removes the component; however, I wasn't confident enough about those changes to rule out possible side effects.

That said, it's probably worth trying and if we run into any trouble I suppose we can always revise this after the fact. Opened https://github.com/mongodb/mongo-php-driver/pull/1567 with some refactoring, and if that still passes I'll come back and refactor this PR.

Reply:
https://github.com/mongodb/mongo-php-driver/pull/1567 looks good, so I've updated this PR to use a single, consolidated step.